### PR TITLE
Include `commit_message` body in `pr_body`

### PR DESCRIPTION
**Pull request by jakirkham**
Improve visibility of additional details from the `commit_message` by including in the migrator PR's body message.

### CODE REVIEW

**Comment by jakirkham** [comment]
FWIW usage like this case ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4527 ) is one situation that might benefit from this added visibility.

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

File: conda_forge_tick/migrators/migration_yaml.py [hunk 1]
```diff
@@ -301,11 +301,22 @@ def pr_body(self, feedstock_ctx: "FeedstockContext") -> str:
                 )
             )
 
+        commit_body = " ".join(
+            [e for e in self.commit_message(feedstock_ctx).splitlines()[1:] if e],
+        )
+        if commit_body:
+            additional_body += (
+                "<hr>"
```
Comment (jakirkham) [inline conda_forge_tick/migrators/migration_yaml.py:309]:
```suggestion
                "<hr>\n\n"
```

File: conda_forge_tick/migrators/migration_yaml.py [hunk 2]
```diff
@@ -301,11 +301,22 @@ def pr_body(self, feedstock_ctx: "FeedstockContext") -> str:
...
+                "Here are some more details about this specific migrator:\n\n"
+                "{commit_body}\n\n"
+            ).format(commit_body=commit_body)
+
         children = "\n".join(
             [" - %s" % ch for ch in self.downstream_children(feedstock_ctx)],
         )
         if len(children) > 0:
             additional_body += (
+                "<hr>"
```
Comment (jakirkham) [inline conda_forge_tick/migrators/migration_yaml.py:319]:
```suggestion
                "<hr>\n\n"
```

**Review by jakirkham — COMMENTED** [review]
Sorry missed some newlines below. Fixing with PR ( https://github.com/regro/cf-scripts/pull/1714 )

**Comment by carterbox** [comment]
Nice! This is what I expected to happen in the first place because GitHub does this when you open a PR for a branch with a single commit. Thanks, @jakirkham!

**Comment by jakirkham** [comment]
Happy to help 🙂

Yeah it may be missed as we push 2 commits with these migrators or because the PR body is set

In any event think your PR adding the commit message got at the core issue 🙏

The rest of this is just tinkering around the edges 😉

<hr>

FWIW there's an example of what this looks like here: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4527#issuecomment-1592250672

We can probably tweak it more. Though it does feel like we've got the basic pieces in place 🙂

**Pull request merged**